### ISSUE TITLE
Add support for custom settings

### ISSUE THREAD

**Issue by spatialdude** [XLSForm/pyxform#66]
Similar to how #65 discusses the ability to add custom namespaces to support custom `bind::` and `body::` properties for questions, it would be very useful to be able to add custom global properties in the **settings** sheet.

e.g.  The column `head::esri::somethingglobal` in **settings** containing the value `The Value` might translate into the xml as 

``` xml
<?xml version="1.0"?>
<h:html xmlns="http://www.w3.org/2002/xforms" xmlns:esri="http://esri.com/xforms" xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:jr="http://openrosa.org/javarosa" xmlns:orx="http://openrosa.org/xforms/" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
  <h:head>
    <h:title>My Survey</h:title>
    <esri:somethingglobal>The value</esri:somethingglobal>

   ...
```

**Comment by MartijnR** [comment]
A feature like this make sense to me for use cases where the style column is not appropriate.

Commcare seems to have also looked into this as well. Their approach is to add secondary (external) instances. I think I like that approach better as the submitted record does not include these readonly variables. It's nicely separated.  A namespaced internal secondary instance or a jr://-uri-non-namespaced external secondary instance (like [these](http://dimagi.github.io/xform-spec/#uri-support)) would both seem appropriate to me.

What do you think?

P.S. not immediately clear how to describe this in XLSForm, but we can figure that out after the XForm syntax.

**Comment by lognaturel** [comment]
Is this now addressed by external instance support #107? 

I do think it would be nicer if instances could be generated from the choices sheet and internal secondary instances could be specified as I mentioned at https://github.com/XLSForm/pyxform/issues/107#issuecomment-353414375 Are those ready to file as issues or should there be more discussion around them?

**Comment by MartijnR** [comment]
> Are those ready to file as issues or should there be more discussion around them?

I think so.

I also think this issue can be closed now.
